src/indexCors.ts

```diff
@@ -31,8 +31,12 @@ app.all('*', async (req: Request, res: Response) => {
     headers: {...headers,
       'X-Forwarded-For': ipString,
       'Forwarded': `for=${ipString}`
-    } as { [key: string]: string },
-    body: JSON.stringify(body)
+    } as { [key: string]: string }
+  }
+
+
+  if (method !== 'GET' && method !== 'HEAD') {
+    init.body = JSON.stringify(body)
```

Comment:
Are these really the only methods that don't have a request body? We should choose white-listing or black-listing strategies based on whichever is more concise. Missing `OPTIONS`, `TRACE`, and `CONNECT`. It may be redundant for the purpose of use within web-views, but better to be safe.

Reply:
DELETE "may" have a body, so I guess it's okay to have a policy to always have a body for those. https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE

Reply:
The error generated when you put a body when non needed is that GET and HEAD must not have a body.

Reply:
What about OPTIONS, TRACE or CONNECT? Will these methods also error if given a body?

Reply:
I'll make the change to not pass a body if the request has an empty object for `body`